pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoader.java

```diff
@@ -168,8 +168,13 @@ void loadMailingListData(Handle handle, JsonElement data, String studyCode) {
             firstName = getStringValueFromElement(thisEl, "firstname");
             lastName = getStringValueFromElement(thisEl, "lastname");
             email = getStringValueFromElement(thisEl, "email");
-            dao.insertByStudyGuidIfNotStoredAlready(firstName, lastName, email,
-                    studyCode, null, dateCreatedMillis);
+            if (StringUtils.isBlank(firstName)) {
+                firstName = " ";
```

Comment:
Can the name be empty string, or does it have to be a space?

Reply:
as long as it passes DB NOT NULL constraint, we are good. 
updated to empty string ""